Sources/PagecallSDK/NativeBridge.swift

```diff
@@ -144,6 +166,18 @@ class NativeBridge: Equatable {
             } else {
                 respond(PagecallError.other(message: "Failed to getPermissions"), nil)
             }
+        case .getMediaStats:
+            if let miController = mediaController as? MiController {
+                let jsonString = miController.getMediaStats()
+                switch parseMediaStats(jsonString: jsonString) {
+                    case .success(let stat):
+                        respond(nil, try? JSONEncoder().encode(stat))
+                    case .failure(let error):
+                        respond(nil, try? JSONEncoder().encode(NullEncodable()))
+                }
+            } else {
+                respond(nil, try? JSONEncoder().encode(NullEncodable()))
```

Comment:
If we were to respond with errors in this manner, it seems that it would offer the advantage of easily identifying and addressing any subsequent exceptional situations.

```suggestion
                        respond(error)
                }
            } else {
                respond(PagecallError.other(message: "Not supported in this environment")
```